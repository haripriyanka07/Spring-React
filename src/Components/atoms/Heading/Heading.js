import { Typography } from "@material-ui/core";

const Heading = (props) => {
    return (
        <Typography align='center' data-testid="h1tag" variant='h4'>{props.children}</Typography>
    )
}
export default Heading;

