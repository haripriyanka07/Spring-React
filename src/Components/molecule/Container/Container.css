.root{
    margin-left: auto;
    margin-right: auto;
    width: 65%;
    padding-top: 10%;
}