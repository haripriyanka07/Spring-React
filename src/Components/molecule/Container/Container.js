import { Grid } from '@material-ui/core'
import "./Container.css"

const Container = (props) => {
    return (
        <div >
            <Grid container spacing={0} direction="column" alignItems="center" justify="center" className="root">
                    {props.children}
            </Grid>
        </div>
    )
}
export default Container;