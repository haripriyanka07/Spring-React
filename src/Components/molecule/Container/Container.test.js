import React from 'react';
import Container from './Container';
import renderer from 'react-test-renderer';
import Heading from '../../atoms/Heading/Heading';

it('test the Container', () => {
    const asFragment = renderer.create(<Container><Heading>Heading</Heading></Container>).toJSON();
    expect(asFragment).toMatchSnapshot()
})

