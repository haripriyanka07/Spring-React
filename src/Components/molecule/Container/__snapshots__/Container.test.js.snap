// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the Container 1`] = `
<div>
  <div
    className="MuiGrid-root root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4 MuiTypography-alignCenter"
      data-testid="h1tag"
    >
      Heading
    </h4>
  </div>
</div>
`;
